### ISSUE TITLE
Wayland: Multi monitor video playback slows window update rate when one window is not shown

### ISSUE THREAD

**Issue by josephtheengineer** [qutebrowser/qutebrowser#5482]
**Version info**:

qutebrowser v1.12.0
Git commit: 
Backend: QtWebEngine (Chromium 69.0.3497.128)
Qt: 5.12.7
Window Manager: SwayWM
Distro: NixOS
Graphics: Intel i5-7400 Internal graphics

**Does the bug happen if you start with `--temp-basedir`?**
~~Yes~~ No
Only works when two windows share the same basedir (session)

**Description**
All windows seem to slow down their refresh rate even though one is still showing. The other window seems to slow down it's refresh rate alot (1 sec per frame). 

Could be a bug in SwayWM though (but does not happen with Firefox).

[Video demonstration](https://youtu.be/5opuiNgkJqE)

**How to reproduce**

1. Open two Qutebrowser windows and place them on two seperate monitors.
2. Play YouTube videos in both windows
3. Switch away from one window (move to a different workspace)

**Comment by The-Compiler** [comment]
Can you check with another QtWebEngine-based browser such as KDE Falkon?

**Comment by josephtheengineer** [comment]
Yep, can confirm that it also happens in KDE Falkon v3.1.0 (QtWebEngine 5.12.7). Only lags when the other tab is not the shown tab.

**Comment by josephtheengineer** [comment]
Also just realised this does not happen with --temp-basedir although it still happens without any configs set.

**Comment by The-Compiler** [comment]
How exactly are you opening the second window with `--temp-basedir`?

**Comment by josephtheengineer** [comment]
Doesn't happen:
```
Terminal 1:
zsh> qutebrowser --temp-basedir
Terminal 2:
zsh> qutebrowser --temp-basedir
```

Happens:
```
mv ~/local/lib/qutebrowser ~/desktop
mv ~/local/etc/qutebrowser ~/desktop
Terminal 1:
zsh> qutebrowser
Terminal 2:
zsh> qutebrowser
```

**Comment by The-Compiler** [comment]
So I assume if you use `--temp-basedir` and `:open -w` for the second window, you see it as well?

**Comment by josephtheengineer** [comment]
Hmmm doesn't seem to lag with that.

**Comment by The-Compiler** [comment]
Odd - but given that it happens in Falkon as well, this seems like #4692 / [QTBUG-69077](https://bugreports.qt.io/browse/QTBUG-69077) - nothing qutebrowser can do about that as far as I'm aware. Even though the original fix should have made it to Qt 5.12.4, you might want to check how things are looking with more recent Qt versions.
